packages/react-core/src/components/Nav/NavList.tsx

```diff
@@ -143,18 +143,16 @@ class NavList extends React.Component<NavListProps> {
           <PageSidebarContext.Consumer>
             {({ isSidebarOpen }) => (
               <React.Fragment>
-                {isHorizontal && (
+                {isHorizontal && (!scrollViewAtStart || !scrollViewAtEnd) && (
                   <div className={css(styles.navScrollButton)}>
                     <Button
                       variant="plain"
                       aria-label={backScrollAriaLabel || ariaLeftScroll}
                       onClick={this.scrollBack}
-                      disabled={scrollViewAtStart}
+                      isDisabled={scrollViewAtStart}
                       tabIndex={isSidebarOpen ? null : -1}
                     >
-                      <span className={css(styles.buttonIcon)}>
-                        <AngleLeftIcon />
-                      </span>
+                      <AngleLeftIcon />
```

Comment:
Looks like we need the buttonIcon wrapper here to resolve the issue Mark noted above.